Code/PgSQL/rdkit/expected/rdkit-91.out

```diff
@@ -1583,3 +1583,140 @@ M  END'));
  
 (1 row)
 
+-- sanitization and Hs: do not sanitize, removing Hs has no effect
+select mol_to_smiles(mol_from_ctab('
+     RDKit          2D
+
+  0  0  0  0  0  0  0  0  0  0999 V3000
+M  V30 BEGIN CTAB
+M  V30 COUNTS 3 2 0 0 0
+M  V30 BEGIN ATOM
+M  V30 1 C 0.000000 0.000000 0.000000 0
+M  V30 2 F 1.299038 0.750000 0.000000 0
+M  V30 3 H 2.598076 -0.000000 0.000000 0
+M  V30 END ATOM
+M  V30 BEGIN BOND
+M  V30 1 1 1 2
+M  V30 2 1 2 3
+M  V30 END BOND
+M  V30 END CTAB
+M  END
+',false,false,true));
+ mol_to_smiles 
+---------------
+ [H]FC
+(1 row)
+
+-- sanitization and Hs: do not sanitize, do not remove Hs
+select mol_to_smiles(mol_from_ctab('
+     RDKit          2D
+
+  0  0  0  0  0  0  0  0  0  0999 V3000
+M  V30 BEGIN CTAB
+M  V30 COUNTS 3 2 0 0 0
+M  V30 BEGIN ATOM
+M  V30 1 C 0.000000 0.000000 0.000000 0
+M  V30 2 F 1.299038 0.750000 0.000000 0
+M  V30 3 H 2.598076 -0.000000 0.000000 0
+M  V30 END ATOM
+M  V30 BEGIN BOND
+M  V30 1 1 1 2
+M  V30 2 1 2 3
+M  V30 END BOND
+M  V30 END CTAB
+M  END
+',false,false,false));
+ mol_to_smiles 
+---------------
+ [H]FC
+(1 row)
+
+-- sanitization and Hs: sanitize, do not remove Hs
+select mol_to_smiles(mol_from_ctab('
+     RDKit          2D
+
+  0  0  0  0  0  0  0  0  0  0999 V3000
+M  V30 BEGIN CTAB
+M  V30 COUNTS 3 2 0 0 0
+M  V30 BEGIN ATOM
+M  V30 1 C 0.000000 0.000000 0.000000 0
+M  V30 2 O 1.299038 0.750000 0.000000 0
+M  V30 3 H 2.598076 -0.000000 0.000000 0
+M  V30 END ATOM
+M  V30 BEGIN BOND
+M  V30 1 1 1 2
+M  V30 2 1 2 3
+M  V30 END BOND
+M  V30 END CTAB
+M  END
+',false,true,false));
+ mol_to_smiles 
+---------------
+ [H]OC
+(1 row)
+
+-- sanitization and Hs: default is to sanitize and remove Hs
+select mol_to_smiles(mol_from_ctab('
+     RDKit          2D
+
+  0  0  0  0  0  0  0  0  0  0999 V3000
+M  V30 BEGIN CTAB
+M  V30 COUNTS 3 2 0 0 0
+M  V30 BEGIN ATOM
+M  V30 1 C 0.000000 0.000000 0.000000 0
+M  V30 2 O 1.299038 0.750000 0.000000 0
+M  V30 3 H 2.598076 -0.000000 0.000000 0
+M  V30 END ATOM
+M  V30 BEGIN BOND
+M  V30 1 1 1 2
+M  V30 2 1 2 3
+M  V30 END BOND
+M  V30 END CTAB
+M  END
+',false));
+ mol_to_smiles 
+---------------
+ CO
+(1 row)
+
+-- sanitization and Hs: sanitize, do not remove Hs, input coords manipulated
```

Comment:
I admit I am not sure about where the manipulation of the input coordinates is implemented in this case, but this issue affects the reviewer, not the PR.

Reply:
> I admit I am not sure about where the manipulation of the input coordinates is implemented in this case, but this issue affects the reviewer, not the PR.

It's checking that the input conformer isn't actually stored by verifying that the output coordinates from `mol_to_v3kctab()` do not match the coordinates provided.

Reply:
Ah, of course.. thanks, I misunderstood the comment and thought it implied that coords were explicitly manipulated because of the sanitization + not removing the Hs, which was unexpected to me and I then didn't think about the `keep_conformer` parameter.